inbound-app/app/globals.css

```
@@ -22,7 +22,7 @@
     --destructive-foreground: 98% 0.02 240;
     --border: 92% 0.02 240;
     --input: 92% 0.02 240;
-    --ring: 15% 0.05 264;
+    --primary: 25% 0.08 264;
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--primary is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)